src/data/commands.js

```diff
@@ -19,3 +20,50 @@ export function load(dataName) {
         chatScene.writeText('The test data "' + dataName + '" does not exist.');
     }
 }
+
+export function unlockLevel(dataName) {
+    let chatScene = characterData.getScene(CONSTANTS.SCENES.CHAT);
+
+    const constName = dataName.toUpperCase();
+    if (constName == "ALL") {
+        // unlock all levels
+        for (let level in characterData.characterData.levels) {
+            characterData.setQuestCompleted(CONSTANTS.PREREQUISITES[level]);
+        }
+    } else if (characterData.characterData.levels.hasOwnProperty(constName)) {
```

Comment:
Any reason to use hasOwnProperty instead of bracket notation?

Reply:
Not sure what alternative you're suggesting.

Reply:
```suggestion
    } else if (characterData.characterData.levels[constName]) {
```

Reply:
I may be getting old and crotchety, but I prefer the explicit hasOwnProperty. The brackets leave some ambiguity as to whether it is checking for existence or whether it is a boolean value

Reply:
You are but that’s fine